src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/StartlistPage.razor

```diff
@@ -0,0 +1,32 @@
+﻿@using NTS.Domain.Core.Objects
+@using NTS.Judge.Blazor.Ports
+@using System.Collections
+@using Not.Blazor.TM
+@using Not.Extensions
+
+@attribute [Route(STARTLIST_PAGE)]
+@inherits NotComponent
+
+
+<MudTabs Elevation="2" Rounded="true" ApplyEffectsToContainer="true" PanelClass="pa-6">
+    <MudTabPanel Text="Upcoming Starts">
+        <UpcomingStarts Items="UpcomingStartsList"></UpcomingStarts>
+    </MudTabPanel>
+    <MudTabPanel Text="Start History">
+        <StartlistHistory Items="StartHistoryList"></StartlistHistory>
+    </MudTabPanel>
+</MudTabs>
```

Comment:
Tabs within tabs doesn't look good. Use `NotPage` to conform to Dashboard pages.